resolwe/test/testcases/process.py

```diff
@@ -535,6 +538,27 @@ def assertStatus(self, obj, status):  # pylint: disable=invalid-name
             msg="Data status is '{}', not '{}'".format(obj.status, status) + self._debug_info(obj)
         )
 
+    def _get_output_field(self, obj, path):
+        """Return object's output field value and schema.
+
+        :param obj: object with the output field
+        :type obj: ~resolwe.flow.models.Data
+
+        :param str path: path to
+            :class:`~resolwe.flow.models.Data` object's output field
+
+        """
+        field_schema, field = None, None
+        for field_schema, field, field_path in iterate_fields(obj.output, obj.process.output_schema, ''):
```

Comment:
This is an analog of iteration in [`assertFields`](https://github.com/genialis/resolwe/blob/master/resolwe/test/testcases/process.py#L555). If `path_prefix` is left set to `None`, we cannot make the comparison in the next line because in that case [`field_path` is not returned by `iterate_fields`](https://github.com/genialis/resolwe/blob/master/resolwe/flow/utils/iterators.py#L32).